usr/share/rear/backup/DUPLICITY/default/500_make_duplicity_backup.sh

```diff
@@ -21,41 +21,37 @@ if [ "$BACKUP_PROG" = "duplicity" ] ; then
     # by falk hoeppner
 
     if [ -n "$BACKUP_DUPLICITY_ASK_PASSPHRASE" ]; then
-        LogPrint "Warning !
-    BACKUP_DUPLICITY_ASK_PASSPHRASE set, The Passphrase needs to be provided Interactively on Restore."
+        LogPrint "Warning: BACKUP_DUPLICITY_ASK_PASSPHRASE set, The Passphrase needs to be provided Interactively on Restore."
     fi
 
     LogPrint "Creating $BACKUP_PROG archives on '$BACKUP_DUPLICITY_URL'"
 
     # todo: check parameters
     BKP_URL="$BACKUP_DUPLICITY_URL"
-    
+
     DUP_OPTIONS="$BACKUP_DUPLICITY_OPTIONS"
 
+    if [ -n "$BACKUP_DUPLICITY_FULL_IF_OLDER_THAN" ]; then
+        FULL_BACKUP="--full-if-older-than $BACKUP_DUPLICITY_FULL_IF_OLDER_THAN"
+    fi
+
     if [ -n "$BACKUP_DUPLICITY_GPG_ENC_KEY" ]; then
         GPG_KEY="--encrypt-key $BACKUP_DUPLICITY_GPG_ENC_KEY"
     fi
     PASSPHRASE="$BACKUP_DUPLICITY_GPG_ENC_PASSPHRASE"
 
+    if [ -z "$BACKUP_DUPLICITY_EXCLUDE_EVALUATE_BY_SHELL" ]; then
+        set -f # Temporarily Stop Evaluation of Patterns By the Shell
+    fi
 
-    # EXCLUDES="${TMP_DIR}/backup_exclude.lst"
-
-    # NMBRS=${#BACKUP_DUPLICITY_EXCLUDE[$@]}
-    # echo NMBRS = $NMBRS
-
-    # for i in $(seq 0 $(($NMBRS - 1)) )
-    # do
-    #     LogPrint "Exclude No $i = ${BACKUP_DUPLICITY_EXCLUDE[$i]}"
-    #     echo "${BACKUP_DUPLICITY_EXCLUDE[$i]}" >> "${EXCLUDES}"
-    # done
-
-    # runs without external file, but all the * in the excludelist
-    # will expanded :-(
-    #
     for EXDIR in ${BACKUP_DUPLICITY_EXCLUDE[@]} ; do
         EXCLUDES="$EXCLUDES --exclude $EXDIR"
     done
 
+    if [ -z "$BACKUP_DUPLICITY_EXCLUDE_EVALUATE_BY_SHELL" ]; then
```

Comment:
I think BACKUP_DUPLICITY_EXCLUDE_EVALUATE_BY_SHELL is a boolean variable
so that `is_true` or `is_false` shold be used.